package.json

```diff
@@ -41,10 +41,10 @@
     "request": "^2.88.0",
     "rimraf": "^3.0.0",
     "semver": "^6.3.0",
-    "stylelint": "^10.1.0"
+    "stylelint": ">=10.1.0"
```

Comment:
I went with this syntax but could see updating this to `^11.0.0`.

Reply:
Not a big thing, but would be better to use `^11.0.0` for consistency.

Reply:
Updated in this commit: https://github.com/kristerkari/stylelint-scss/pull/369/commits/830a85a659c59ac27969e8fab6b6bca8a13cac54